task_DEV-9/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.IO;
+using System.Collections.Generic;
+using System;
+
+namespace task_DEV_9
+{
+  // Current program reads two strings from file and change random char sequence
+  // from one string to another random char sequence from the second string.
+  class Program
+  {
+    static void Main(string[] args)
+    {
+      string[] dataFromFile = null;
+      try
+      {
+        dataFromFile = new FileReader().GetStringsFromCustomFile();
+      }
+      catch (IndexOutOfRangeException ex)

```

Comment:
Use one catch for all exceptions, you only log info, no need in separate processing